Praat/PraatVersionFixer.py

```diff
@@ -44,7 +44,11 @@ class PraatVersionFixer(Processor):
     def read_bundle_info(self, path):
         """Read Contents/Info.plist inside a bundle."""
         plistpath = os.path.join(path, "Contents", "Info.plist")
-        info, _, error = NSPropertyListSerialization.propertyListFromData_mutabilityOption_format_errorDescription_(  # noqa
+        (
+            info,
+            _,
+            error,
+        ) = NSPropertyListSerialization.propertyListFromData_mutabilityOption_format_errorDescription_(  # noqa
```

Comment:
Naw. We should murder this. There's absolutely no reason not to just use plistlib.